fdmi/plugins/fdmi_app

```diff
@@ -103,10 +205,6 @@ def process_fdmi_record(kv_record, p):
         'cr_val': cr_val
     }
 
-    ri = bytes('{}\n'.format(rec_id), encoding = "utf-8")
```

Comment:
Just to understand correctly, instead of writing to a pipe, writing to kafka, is it correct?

Reply:
Please see my comments above.